scripts/west_commands/tests/test_mdb.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2020 Synopsys
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+from os import path
+from unittest.mock import patch
+from unittest.mock import call
+
+import pytest
+
+from runners.mdb import MdbBinaryRunner
+from conftest import RC_KERNEL_ELF, RC_BOARD_DIR
+
+
```

Comment:
Yet another double newline? I start feeling it's really important :)

Reply:
Some python style guides require two newlines between sections, and you'll see both styles. They are fine as long as it's consistent within a file.